README.md

```diff
@@ -1,3 +1,83 @@
 # Oragono Docker
 
-This repo holds Oragono's Dockerfiles and such.
+This repo holds [Oragono](https://github.com/oragono/oragono)'s Dockerfiles and
+related materials.
+
+## Tags
+
+The following tags are available:
+
+ * `dev` - latest development version; published periodically (may be unstable)
```

Comment:
Should we mention that `latest` is the latest stable version of Oragono here as well?

Reply:
Oh saw your note about the v1.0.0 tag and stable. Yeah let's wait for now in that case, thanks!